### ISSUE TITLE
AlphaFold: Using AI for scientific discovery

### ISSUE THREAD

**Issue by lykaust15** [Benjamin-Lee/deep-rules#166]
https://deepmind.com/blog/alphafold/

Currently, there is no formal paper introducing AlphaFold available. But as a paper in 2019, I think it is worth mentioning AlphaFold in the paper.

**Comment by AlexanderTitus** [comment]
This is a very interesting application, but I don't see how it would directly add to these recommendations. This is not a formal review, but a concise set of recommendations. 

Can you suggest how it would fit?

**Comment by lykaust15** [comment]
Agree on your point that this is not a formal review paper.

But shall we include some interesting applications of deep learning in biology in the introduction part? Just like:
```
In many cases, novel biological insights have been revealed through careful evaluation of DL methods ranging from predicting protein-drug binding kinetics [2] to identifying the lab-of-origin of synthetic DNA [3]. 
```
But since there have been examples here, not sure whether we should further include AlphaFold. 
On the other hand, people would not say AlphaFold is not interesting.

**Comment by AlexanderTitus** [comment]
Feel free to submit a pull request with the addition and we can take a look. I agree that AlphaFold is a great application and could be good to include.

**Comment by rasbt** [comment]
Hm, I agree that it's not really fitting into the context regarding DL recommendations for biologists, but it could fit into the intro as an example of  interesting and challenging problems being tackled by DL.

Instead of referring to AlphaFold, we could actually cite some of the original work that AlphaFold seems to have copied and extended, e.g., 

- Wang, S., Sun, S., Li, Z., Zhang, R., & Xu, J. (2017). Accurate de novo prediction of protein contact map by ultra-deep learning model. PLoS computational biology, 13(1), e1005324.

**Comment by SonjaAits** [comment]
I agree it would be very important to name structural biology as one of the example domains where DL has huge potential. The AlphaFold made the international mainstream news, after all. It can be cited like this until the final publication is released (according to their website https://deepmind.com/blog/alphafold/): 
De novo structure prediction with deep-learning based scoring
R.Evans,  J.Jumper, J.Kirkpatrick, L.Sifre, T.F.G.Green, C.Qin, A.Zidek, A.Nelson, A.Bridgland, H.Penedones, S.Petersen, K.Simonyan, S.Crossan, D.T.Jones, D.Silver, K.Kavukcuoglu, D.Hassabis, A.W.Senior
In Thirteenth Critical Assessment of Techniques for Protein Structure Prediction (Abstracts) 1-4 December 2018.
